spec/acceptance/machine_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'spec_helper_acceptance'
+require 'spec_helper_acceptance_local'
```

Comment:
this should not be needed, as `spec_helper_acceptance` is already requiring this